test/e2e/volumes.go

```diff
@@ -934,7 +933,7 @@ var _ = framework.KubeDescribe("Volumes [Feature:Volumes]", func() {
 	// vSphere
 	////////////////////////////////////////////////////////////////////////
 
-	framework.KubeDescribe("vsphere", func() {
+	framework.KubeDescribe("vsphere [Feature:Volumes]", func() {
```

Comment:
@divyenpatel As far as I understand, the string after the ":" is ignored by CI testing, so yes.
Related to this, in one of my reviews of a vsphere e2e test pr, I asked you to remove the `[Feature:Volumes]` because it was only used in _volumes.go_. Since then I have learned more about this tag. It is ignored for pr merging and it causes a seprate test suite to be used for release testing (not ignored there).